assets/batch_score/components/driver/src/batch_score_oss/common/configuration/command_line_argument_specification.py

```diff
@@ -55,11 +54,8 @@
         'choices': [
             ApiType.Completion,
             ApiType.ChatCompletion,
-            ApiType.Embedding,
-            ApiType.Vesta,
-            ApiType.VestaChatCompletion,
         ],
-        'default': ApiType.Completion,
+        # 'default': ApiType.Completion,
```

Comment:
Why is this commented out? Do you want to remove this or switch to `ApiType.ChatCompletion` if that fits your case better?

Reply:
Good point, I had commented it out for one test scenario. But forgot to uncomment it out. Uncommented